packages/devtools_app/lib/src/screens/inspector_v2/inspector_data_models.dart

```diff
@@ -91,6 +91,26 @@ List<double> computeRenderSizes({
   return renderSizes;
 }
 
+/// Data pattern containing a widget's widths or heights.
+typedef WidgetSizes = ({
+  // Whether this record represents a widget's widths or heights.
+  SizeType type,
+
+  /// Either the widget's left or top padding.
+  double paddingA,
+
+  // Either a widget's width or height.
+  double widgetSize,
+
+  // Either a widget's right or bottom padding.
```

Comment:
same here. Can you also expand on the dart doc to specify when these values are left vs top, right vs bottom, width vs height depending on whether `type` is `SizeType.widths` vs `SizeType.heights`

Reply:
done!